### DISCUSSION TITLE
Can't find Images

### DISCUSSION THREAD

**Discussion by Knutsford** [dompdf/dompdf#2643] [Q&A]
```
namespace Dompdf;
require_once 'dompdf/dompdf/autoload.inc.php';

$dompdf = new Dompdf(); 

$dompdf->loadHtml('

<img src="http://www.blaablaa.uk/PDF/FOOTER.jpg" width="100%">

');
$dompdf->set_option('isRemoteEnabled', true);

$dompdf->setPaper('A4', 'portrait');
$dompdf->render();
$dompdf->stream("",array("Attachment" => false));
```

I am getting image not found or type unknown even though that is the correct url. All the other images have the same problem. Any idea please - thanks

**Comment by bsweeney** [comment]
Any errors reported by PHP? Only thing I can think off is maybe Dompdf does not have access to the temp directory Dompdf uses the `sys_get_temp_dir()` value by default. You can change the value by setting the "tempDir" option.

**Comment by Knutsford** [comment]
The site is pasword protected at the moment as it is a test one. Could it be that but I am logged in behind the protection?

**Reply by bsweeney** [reply]
That's probably what's going on. Though you yourself may be logged in Dompdf uses it's own context when connecting to the site. You're in luck, however, as we just added documentation to the wiki related to [customizing the context](https://github.com/dompdf/dompdf/wiki/Common-Use-Cases#custom-http-stream-context). If you're using basic authentication then the example provided will be sufficient.

**Comment by Knutsford** [comment]
Brilliant - that worked - phew
